sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/BuilderHelper.java

```diff
@@ -210,4 +211,8 @@ private static Tracer createTracer(ClientOptions clientOptions) {
         return TracerProvider.getDefaultProvider()
             .createTracer(CLIENT_NAME, CLIENT_VERSION, TABLES_TRACING_NAMESPACE_VALUE, tracingOptions);
     }
+
+    private static TableAudience getDefaulTableAudience(boolean isCosmosEndpoint) {
```

Comment:
I don't think this is used anywhere? Or, was this copied from somewhere and now is private to this class?

Reply:
It's used on line 67, it replaces the nested ternary operator for readability.

Reply:
Ah, haha, missed that. Looks good than!